Gruntfile.js

```diff
@@ -0,0 +1,30 @@
+/* Wrapper function */
+module.exports = function(grunt){
+
+  /* Initialize  grunt tasks */
+  grunt.initConfig({
+    /* Importing metadata from package.json */
+    pkg: grunt.file.readJSON('package.json'),
+
+    /* Uglify with concat & sourceMap */
+    uglify: {
+      options: {
+        mangle: false,
```

Comment:
Needed for minifying angular code so that $scope and angular specific variable names do not get uglified.